podman/images/__init__.py

```diff
@@ -1,23 +1,23 @@
 """images provides the operations against images for a Podman service."""
 import json
-
 from http import HTTPStatus
+
 import podman.errors as errors
 
 
 def list_images(api):
     """List all images for a Podman service."""
     response = api.get("/images/json")
-    return json.loads(str(response.read(), 'utf-8'))
+    return json.loads(str(response.read(), "utf-8"))
 
 
 def inspect(api, name):
     """Report on named image for a Podman service.
-       Name may also be a image ID.
+    Name may also be a image ID.
```

Comment:
nit of a nit, only change if you've other tweaks.
```suggestion
    Name may also be an image ID.
```

Reply:
Actually, this is user-facing, can you clean this and I'll LGTM it?